test/Libraries/Microsoft.Extensions.AI.OpenAI.Tests/OpenAIResponseClientIntegrationTests.cs

```diff
@@ -315,7 +315,7 @@ public async Task GetStreamingResponseAsync_BackgroundResponses_StreamResumption
 
         int updateNumber = 0;
         string responseText = "";
-        object? continuationToken = null;
+        ResponseContinuationToken? continuationToken = null;
```

Comment:
I'm surprised more changes in the tests weren't necessary. Does that mean we're lacking tests? Or that there are now lots of unnecessary casts in those tests? Or something else?